Tests/LibWeb/Text/input/js-float16.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<script src="include.js"></script>
+<script>
+    test(() => {
+        // From a length
+        const float16 = new Float16Array(2);
```

Comment:
This test seems nice, but should probably be in a LibJS test instead of a LibWeb one.

Possibly somewhere like https://github.com/LadybirdBrowser/ladybird/tree/39f844f77c719b5c7789d644aac6ea44c87b9625/Userland/Libraries/LibJS/Tests/builtins/TypedArray ?

Going down that route, one commit to add the type to LibJS and another one to add LibWeb integration seems appropriate.